### PR TITLE
channel switcher refinements

### PR DESCRIPTION
**Pull request by patosullivan**
https://github.com/tloncorp/tlon-apps/assets/1221094/f281287d-f781-47e6-b76a-1dc6476a6301

Fixes LAND-1709

### CODE REVIEW

File: packages/ui/src/components/ChannelNavSections.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  channels,
+  onSelect,
+  paddingBottom,
+}: {
+  group: db.GroupWithRelations;
+  channels: db.ChannelWithLastPost[];
+  onSelect: (channel: any) => void;
+  paddingBottom?: number;
+}) {
+  const hasSomeUnGroupedChannels = useMemo(
```
Comment (dnbrwstr) [inline packages/ui/src/components/ChannelNavSections.tsx:19]: Could this just be a check on the length of `ungroupedChannels`?

**Review by dnbrwstr — APPROVED** [review]
Looking good, just some very minor nitpicks.

File: packages/ui/src/components/ChannelNavSection.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          return 'ChannelGalleries';
+        default:
+          return 'ChannelTalk';
+      }
+    },
+    [getChannel]
+  );
+
+  return (
+    <YGroup.Item key={section.id}>
```
[the remark this answers is not in the document]
Comment (patosullivan) [reply packages/ui/src/components/ChannelNavSection.tsx]: Yeah, good point.

File: packages/ui/src/components/ChannelNavSections.tsx (hunk 1 above)
Comment (patosullivan) [reply packages/ui/src/components/ChannelNavSections.tsx]: Yep, fixed in latest commit.

**Pull request merged**